docs/errorhandling.rst

```diff
@@ -77,7 +77,7 @@ You might want to show custom error pages to the user when an error occurs.
 This can be done by registering error handlers.
 
 Error handlers are normal :ref:`views` but instead of being registered for
-routes they are registered for exceptions that are rised while trying to
+routes they are registered for exceptions that are raised while trying to
```

Comment:
I agree

On Mon, Aug 29, 2016 at 11:40:17AM -0700, Josh Soref wrote:
> > @@ -77,7 +77,7 @@ You might want to show custom error pages to the user when an error occurs.
> >  This can be done by registering error handlers.
> >  
> >  Error handlers are normal :ref:`views` but instead of being registered for
> > -routes they are registered for exceptions that are rised while trying to
> > +routes they are registered for exceptions that are raised while trying to
> 
> there should probably be a comma after `routes`...
> 
> -- 
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/pallets/flask/pull/1998/files/4d58499bcade7ced03c2d12e7a1248b8027c9941#r76662466